src/JSInterop/Microsoft.JSInterop/ref/Microsoft.JSInterop.netcoreapp3.0.cs

```diff
@@ -27,38 +27,52 @@ public partial interface IJSInProcessRuntime : Microsoft.JSInterop.IJSRuntime
     {
-        protected JSInProcessRuntimeBase() { }
+        protected JSInProcessRuntime() { }
         protected abstract string InvokeJS(string identifier, string argsJson);
         public TValue Invoke<TValue>(string identifier, params object[] args) { throw null; }
     }
+    public static partial class JSInProcessRuntimeExtensions
+    {
+        public static void InvokeVoid(this Microsoft.JSInterop.IJSInProcessRuntime jsRuntime, string identifier, params object[] args) { }
+    }
     [System.AttributeUsageAttribute(System.AttributeTargets.Method, AllowMultiple=true)]
-    public partial class JSInvokableAttribute : System.Attribute
+    public sealed partial class JSInvokableAttribute : System.Attribute
     {
         public JSInvokableAttribute() { }
         public JSInvokableAttribute(string identifier) { }
         public string Identifier { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
     }
-    public static partial class JSRuntime
-    {
-        public static void SetCurrentJSRuntime(Microsoft.JSInterop.IJSRuntime instance) { }
-    }
-    public abstract partial class JSRuntimeBase : Microsoft.JSInterop.IJSRuntime
+    public abstract partial class JSRuntime : Microsoft.JSInterop.IJSRuntime
     {
-        protected JSRuntimeBase() { }
+        protected JSRuntime() { }
         protected System.TimeSpan? DefaultAsyncTimeout { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         protected abstract void BeginInvokeJS(long taskId, string identifier, string argsJson);
         protected internal abstract void EndInvokeDotNet(string callId, bool success, object resultOrError, string assemblyName, string methodIdentifier, long dotNetObjectId);
-        public System.Threading.Tasks.Task<T> InvokeAsync<T>(string identifier, System.Collections.Generic.IEnumerable<object> args, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public System.Threading.Tasks.Task<T> InvokeAsync<T>(string identifier, params object[] args) { throw null; }
+        public System.Threading.Tasks.ValueTask<TValue> InvokeAsync<TValue>(string identifier, object[] args) { throw null; }
+        public System.Threading.Tasks.ValueTask<TValue> InvokeAsync<TValue>(string identifier, System.Threading.CancellationToken cancellationToken, object[] args) { throw null; }
+        public static void SetCurrentJSRuntime(Microsoft.JSInterop.IJSRuntime instance) { }
+    }
+    public static partial class JSRuntimeExtensions
+    {
+        public static System.Threading.Tasks.ValueTask<TValue> InvokeAsync<TValue>(this Microsoft.JSInterop.IJSRuntime jsRuntime, string identifier, params object[] args) { throw null; }
```

Comment:
This is a little strange... is it really possible to provide the same method twice, where one is an extension method but has params?